packages/sw-build/src/index.js

```diff
@@ -0,0 +1,69 @@
+// This will be the logic that powers both module and CLI
+const generateSW = require('./lib/generate-sw');
+const getFileManifestEntries = require('./lib/get-file-manifest-entries');
+const generateFileManifest = require('./lib/generate-file-manifest');
+
+/**
+ * # sw-build
+ *
+ * Use as a command line tool or as a normal node-module.
```

Comment:
Wait, is that true? Or is that just copied/pasted from the old JSDocs? Because I thought the idea was that this was just a node module, not the CLI.